src/components/Bookform.js

```diff
@@ -39,11 +42,14 @@ function Bookform() {
   };
 
   return (
-    <div className="App">
+    <div className="form-container">
+      <h3 className="form-title">ADD NEW BOOK</h3>
       <form onSubmit={handleSubmit}>
-        <input type="text" placeholder="Add book name..." name="title" value={formData.title} onChange={setValue} />
-        <input type="text" placeholder="Add Author's name..." name="author" value={formData.author} onChange={setValue} />
-        <button type="submit">Add Book</button>
+        <div className="form-actions">
+          <input type="text" placeholder="Add book name..." name="title" value={formData.title} onChange={setValue} />
+          <input type="text" placeholder="Add Author's name..." name="author" value={formData.author} onChange={setValue} />
+          <button type="submit">Add Book</button>
+        </div>
       </form>
     </div>
   );
```

Comment:
- [ ] Kindly improve the font of the form component to match the design.

  Link to the design: https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md

  ![jsisiao](https://user-images.githubusercontent.com/54848363/230662793-9da15493-ae23-4fc4-9299-770a84213170.jpg)

Reply:
Ok well noted